### DISCUSSION TITLE
Limiting logs based on severity level

### DISCUSSION THREAD

**Discussion by tballast** [testcontainers/testcontainers-node#436] [Q&A] [answered]
Is there any way to limit the container logs based on whether they are TRACE, DEBUG, INFO, WARN, or ERROR?

Something along the lines of:

`DEBUG=testcontainers:INFO`

or

`DEBUG=testcontainers:exec:error` ?

**Comment by cristianrgreco** [comment, marked as answer]
@tballast I am looking at this again. We use [debug](https://www.npmjs.com/package/debug) for logs. It doesn't have the concept of log levels, it's just really for turning logs off or on. The log levels were introduced just for some extra metadata. If there's some issue with testcontainers I guess it's useful to see all the logs, and if not, they can all be turned off?

`DEBUG=testcontainers:exec:error` doesn't really follow the convention set out by the debug library.

Is there something that is particularly noisy/not useful?
